### PR TITLE
Supported that track can directly be called with DistinctId

### PR DESCRIPTION
**Pull request by henteko**

### CODE REVIEW

**Comment by henteko** [comment]
@tnj Please review it!

File: lib/mengpaneel/tracker.rb [hunk 1]
```diff
@@ -61,6 +61,7 @@ def get_property(property)
       @properties[property]
     end
 
+    alias_method :original_track, :track
     def track(event, properties = {})
```
Comment (tnj) [inline lib/mengpaneel/tracker.rb:65]: `track` can be like

```ruby
track_with_distinct_id @distinct_id, event, properties
```

Comment (henteko) [reply lib/mengpaneel/tracker.rb:65]: thanks! I will fix it!

Comment (henteko) [reply lib/mengpaneel/tracker.rb:65]: @tnj Fixed! Please check again!

**Review by tnj — APPROVED** [review]
LGTM!

**Comment by henteko** [comment]
Thanks!

**Pull request merged**